Purchases/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -104,9 +117,24 @@ class PurchasesOrchestrator {
         }
     }
 
+    func productsFromOptimalStoreKitVersion(withIdentifiers identifiers: [String],
```

Comment:
Stupid comment of the day: should we rename it?
s/`productsFromOptimalStoreKitVersion`/`productsFromNewestAvailableStoreKitFramework`

Reply:
I guess my thinking there was that I'm not sure whether: 
a) the newest will always be the better choice, if sk2 maybe has bugs
b) folks will choose sk2 once a flag is available for them